sources/packages/web/default.conf.template

```diff
@@ -1,5 +1,16 @@
 # nginx default configuration.
+
+log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                  '$status $body_bytes_sent "$http_referer" '
+                  '"$http_user_agent" "$http_x_forwarded_for"';
+
 server {
+  # Log to stdout
+  access_log /dev/stdout main;
```

Comment:
I believe this will print the logs on the console, also after deployment we can check if this is outputting it to kibana.